<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>990</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>990</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>990</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
border-style: outset;
pressed{background-color:rgb(0, 0, 0);border-style: inset; }</string>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
border-style: outset;
pressed{background-color:rgb(0, 0, 0);border-style: inset; }</string>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>691</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 24pt &quot;CabinSketch&quot;;
background-color: rgb(255, 140, 86);
border-radius: 15px; 
border: 10px groove gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>920</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="NewGameButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
border-style: outset;
pressed{background-color:rgb(0, 0, 0);border-style: inset; }</string>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>430</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 32pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
</string>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>520</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>590</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>680</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
</string>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>691</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="L11">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="L12">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="L13">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="L24">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="L23">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="L32">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="L31">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="L21">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="L22">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="L33">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="L34">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="L14">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="L41">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="L42">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="L43">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="L44">
       <property name="styleSheet">
        <string notr="true">color: rgb(208, 104, 0);
border-radius: 20px; 
background-color: rgb(255, 232, 192);
border: 3px groove gray;
font: 75 36pt &quot;CabinSketch&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="playBackProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>880</y>
      <width>1111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 0);
background-color: rgb(255, 244, 224);

font: 75 12pt &quot;CabinSketch&quot;;

</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>Playing Back %p%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>290</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;CabinSketch&quot;;
background-color: rgb(255, 203, 14);
border-radius: 10px; 
border: 10px groove gray;
border-style: outset;
pressed{background-color:rgb(0, 0, 0);border-style: inset; }</string>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Menu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="action_New_Game"/>
    <addaction name="separator"/>
    <addaction name="action_Login"/>
    <addaction name="actionLeader_Board"/>
    <addaction name="separator"/>
    <addaction name="action_Play_Back"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Record"/>
    <addaction name="separator"/>
    <addaction name="actionFail_this_Game"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
   </widget>
   <addaction name="menu_Menu"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Login">
   <property name="text">
    <string>Log &amp;In</string>
   </property>
  </action>
  <action name="actionLeader_Board">
   <property name="text">
    <string>&amp;Leader Board</string>
   </property>
  </action>
  <action name="action_Play_Back">
   <property name="text">
    <string>&amp;Play Back</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log &amp;Out</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionChoose_UserFile">
   <property name="text">
    <string>Choose &amp;UserFile</string>
   </property>
  </action>
  <action name="actionChoose_TraceFile">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionSave_TraceFile">
   <property name="text">
    <string>Save TraceFile</string>
   </property>
  </action>
  <action name="action_New_Game">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
  </action>
  <action name="actionFail_this_Game">
   <property name="text">
    <string>End this Game</string>
   </property>
  </action>
  <action name="action_Record">
   <property name="text">
    <string>&amp;Record</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
